test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -191,6 +192,9 @@ class BalancerServiceImpl : public BalancerService {
         shutdown_(false) {}
 
   Status BalanceLoad(ServerContext* context, Stream* stream) override {
+    // Balancer shouldn't receive the call credentials metadata.
```

Comment:
Let's also check that the backends *do* receive this metadata key.

Reply:
Done.